uaclient/jobs/gcp_auto_attach.py

```diff
@@ -5,51 +5,60 @@
 import logging
 
 from uaclient import config, exceptions
-from uaclient.cli import action_auto_attach
+from uaclient.cli import action_auto_attach, setup_logging
 from uaclient.clouds.gcp import GCP_LICENSES, UAAutoAttachGCPInstance
 from uaclient.clouds.identity import get_cloud_type
 from uaclient.util import get_platform_info
 
 LOG = logging.getLogger(__name__)
 
 
-def gcp_auto_attach(cfg: config.UAConfig) -> None:
+def gcp_auto_attach(cfg: config.UAConfig) -> bool:
```

Comment:
We can drop this right because setup_logging is already called in timer.py:main. @orndorffgrant will have to sort this with his separate PR to ua-license-attach. I don't think you need this right?

Reply:
It's just matching the calls for this timer. I would make it working in this PR, and then @orndorffgrant can adapt to his approach when separate the timer. What do you think, Grant?

Reply:
Whatever is less work since I will be editing this. As long as there is a pattern fir me to follow so that I don't accidentally lose the benefits of this PR

Reply:
Thanks @renanrodrigo for explaining this aspect for inclusion of INFO level message for disabling gcp auto-attach. This makes a lot more sense now for module-specific logging preferences